app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+from typing import List
```

Comment:
The `List` type from the `typing` module is imported but not used correctly in the `feed_animals` function. Ensure that `List` is used with a capital 'L' as it is imported from `typing`.